scripts/stubsabot.py

```diff
@@ -498,32 +500,50 @@ async def create_or_update_pull_request(*, title: str, body: str, branch_name: s
     fork_owner = get_origin_owner()
 
     async with session.post(
-        f"https://api.github.com/repos/{TYPESHED_OWNER}/typeshed/pulls",
+        f"{TYPESHED_API_URL}/pulls",
         json={"title": title, "body": body, "head": f"{fork_owner}:{branch_name}", "base": "main"},
         headers=get_github_api_headers(),
     ) as response:
         resp_json = await response.json()
-        if response.status == 422 and any(
+        if response.status == 201:
+            pr_number = resp_json["number"]
+            assert isinstance(pr_number, int)
+        elif response.status == 422 and any(
```

Comment:
Maybe we could use the `http.HTTPStatus` enum here. I'd fin this much more readable (I can't remember what 201 and 422 mean off the top of my head):

```suggestion
        if response.status == HTTPStatus.CREATED:
            pr_number = resp_json["number"]
            assert isinstance(pr_number, int)
        elif response.status == HTTPStatus.UNAUTHORIZED and any(
```